singularity-eos/eos/eos_gruneisen.hpp

```diff
@@ -400,24 +400,26 @@ PORTABLE_INLINE_FUNCTION Real Gruneisen::BulkModulusFromDensityTemperature(
 template <typename Indexer_t>
 PORTABLE_INLINE_FUNCTION void Gruneisen::DensityEnergyFromPressureTemperature(
     const Real press, const Real temp, Indexer_t &&lambda, Real &rho, Real &sie) const {
-  sie = _Cv * (temp - _T0);
   // We have a branch at rho0, so we need to decide, based on our pressure, whether we
   // should be above or below rho0
-  Real Pref = PressureFromDensityInternalEnergy(_rho0, sie);
+  Real Pref = PressureFromDensityTemperature(_rho0, temp);
```

Comment:
This change isn't substantive since the energy is easily calculated, but I figured this was more consistent